Source/DataSources/EntityView.js

```diff
@@ -314,7 +334,6 @@ function EntityView(entity, scene, ellipsoid) {
   this._mode = undefined;
 
   this._lastCartesian = new Cartesian3();
-  this._defaultOffset3D = undefined;
```

Comment:
This didn't do anything.

Reply:
We usually declare class properties in the constructor so they don't have to be dynamically allocated later by a setter.

Reply:
Sorry, I'm confused. I thought `EntityView._defaultOffset3D` was a _static_ value initialized via the setter by `EntityView.defaultOffset3D = ...` and the `this.defaultOffset3D` referred to here is never read/unused.

In any case, this line doesn't really have anything to do with the PR and I'm happy to put it back in if you want to accept this change.

Reply:
Yeah, I think the idea was to have the property already part of `EntityView` by the time the setter does `this._defaultOffset3D = ...`. @mramato @lilleyse Is this correct?

Reply:
I think @dcervelli's change is correct. The setter is setting a static variable whereas `this._defaultOffset3D = undefined` is an unused `EntityView` instance property.

It looks look it was added in https://github.com/CesiumGS/cesium/commit/d4accd7b993af7a357e4d0eaa76f6f45fd86e33e but I'm not sure why. Do tests still pass?